app/models/task.rb

```diff
@@ -0,0 +1,10 @@
+class Task < ApplicationRecord
+  validates :title, :description, :due_date, presence: true 
+  validate :not_past_date
+
+  def not_past_date
```

Comment:
imho it is not the best name for the method. It doesn't say anything about `due_date` field. Maybe `validate_due_date` or `validate_due_date_not_in_past`?